<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ventana</class>
 <widget class="QMainWindow" name="Ventana">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>327</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ventana</string>
  </property>
  <property name="windowIcon">
   <iconset resource="recursos.qrc">
    <normaloff>:/new/prefix1/iconos/caliente.png</normaloff>:/new/prefix1/iconos/caliente.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Centigrados</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLCDNumber" name="out_cent">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 168, 61);
alternate-background-color: rgb(255, 205, 7);
color: rgb(0, 0, 0);</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDial" name="in_cent">
         <property name="minimum">
          <number>-18</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Fahrenheit</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLCDNumber" name="out_fahr">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 153, 255);
color: rgb(0, 0, 0);</string>
         </property>
         <property name="value" stdset="0">
          <double>32.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDial" name="in_fahr">
         <property name="maximum">
          <number>210</number>
         </property>
         <property name="sliderPosition">
          <number>32</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Kelvin</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLCDNumber" name="out_kelvin">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(170, 85, 255);
color: rgb(0, 0, 0);</string>
         </property>
         <property name="value" stdset="0">
          <double>273.000000000000000</double>
         </property>
         <property name="intValue" stdset="0">
          <number>273</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDial" name="in_kelvin">
         <property name="minimum">
          <number>255</number>
         </property>
         <property name="maximum">
          <number>373</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="recursos.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>in_cent</sender>
   <signal>valueChanged(int)</signal>
   <receiver>out_cent</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>121</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>121</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>in_fahr</sender>
   <signal>valueChanged(int)</signal>
   <receiver>out_fahr</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>351</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>351</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>in_kelvin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>out_kelvin</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>581</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>581</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
